R/execution.r

```diff
@@ -124,9 +152,23 @@ execute = function(request) {
     }
     
     err <<- list()
+    nframe <- NULL
+    tryCatch(evaluate(
+        'stop()',
```

Comment:
jup, that was my original way to find out the stack depth. turns out just doing `nframe <- sys.nframe()` works too